hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/SnapshotChainManager.java

```diff
@@ -131,6 +133,7 @@ private void addSnapshotPath(String snapshotPath,
             new SnapshotChainInfo(snapshotID, prevPathID, null));
 
     // set state variable latestPath snapshot entry to this snapshotID
+    snapshotPathToKey.put(snapshotID, snapTableKey);
     latestPathSnapshotID.put(snapshotPath, snapshotID);
```

Comment:
Comment out of order.

```suggestion
    // store snapshot ID to snapshot DB table key in the map
    snapshotPathToKey.put(snapshotID, snapTableKey);
    // set state variable latestPath snapshot entry to this snapshotID
    latestPathSnapshotID.put(snapshotPath, snapshotID);
```